dockerfiles/logstash-opensearch-sample.conf

```diff
@@ -0,0 +1,17 @@
+# Sample Logstash configuration for creating a simple
+# Beats -> Logstash -> OpenSearch pipeline.
+
+input {
+  beats {
+    port => 5044
+  }
+}
+
+output {
+  opensearch {
+    hosts => ["http://localhost:9200"]
+    index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}"
+    #user => "admin"
```

Comment:
Why are these commented out?

Reply:
In sample i am using http similar to elasticsearch. I commented user here because that will be the default user and password if user switches to https